main/lsp/notifications/did_change_configuration.cc

```diff
@@ -0,0 +1,22 @@
+#include "main/lsp/notifications/did_change_configuration.h"
+#include "main/lsp/LSPPreprocessor.h"
+
+namespace sorbet::realmain::lsp {
+DidChangeConfigurationTask::DidChangeConfigurationTask(const LSPConfiguration &config,
+                                                       std::unique_ptr<DidChangeConfigurationParams> params)
+    : LSPTask(config, LSPMethod::WorkspaceDidChangeConfiguration), params(move(params)) {}
+
+LSPTask::Phase DidChangeConfigurationTask::finalPhase() const {
+    return LSPTask::Phase::RUN;
+}
+
+void DidChangeConfigurationTask::preprocess(LSPPreprocessor &preprocessor) {
+    preprocessor.resume();
+}
+
+void DidChangeConfigurationTask::index(LSPIndexer &indexer) {}
```

Comment:
Do we not need to use this callback to update the indexer's options too?

Reply:
yup